scijava/scijava-struct/src/main/java/org/scijava/struct/Member.java

```diff
@@ -17,7 +17,9 @@
 	String getKey();
 
 	/** Description of the member. */
-	String getDescription();
+	default String getDescription() {
+		return "";
```

Comment:
Are we guaranteeing non-null now? Should put that in the javadoc.

Reply:
Fixed in f69affa4